lib/Serialization/Deserialization.cpp

```diff
@@ -207,12 +207,6 @@ ModuleFile &ModuleFile::getModuleFileForDelayedActions() {
 void ModuleFile::finishPendingActions() {
```

Comment:
I don't think anyone's using this anymore. Can you remove it in a separate commit?